django/pyrog/api/views.py

```diff
@@ -41,13 +41,6 @@ def import_mapping(self, request):
     def export_mapping(self, request, pk=None):
         return self.retrieve(request)
 
-    def get_queryset(self):
```

Comment:
Maybe add an issue to record this issue

Reply:
https://github.com/arkhn/fhir-river/issues/648